### ISSUE TITLE
How to disable the unfurling option in a link?

### ISSUE THREAD

**Issue by frankrod** [bitrise-io/steps-slack-message#7]
I'm using the Install page QR code step along with the slack message step but when I send the link to Slack, I'm seeing the QR code image but I don't want to show the image I just want to share the link without showing that image in slack.

**Comment by viktorbenei** [comment]
There's no option for this right now. Feel free to create a PR if you have the time. If not, then we'll try to schedule it as soon as possible.

Thank you for the feature request!

**Comment by viktorbenei** [comment]
Do you know how this could be achieved with the Slack API? Is it even possible to disable it?

**Comment by kibotu** [comment]
i believe it has something to do with sending images as **attachments.thumb_url** paramenter https://api.slack.com/docs/message-attachments#putting_it_all_together

**Comment by koral--** [comment]
It is definitely possible. Just wrap a URL with backticks: `
URL is linkified but not unfurled:
Online demo: https://api.slack.com/docs/messages/builder?msg=%7B%22text%22%3A%22I%20am%20a%20test%20message%20%60https%3A%2F%2Fbitrise.io%60%22%7D

**Comment by viktorbenei** [comment]
@koral-- that does work with the current version of the step too, right? It should be possible to write it like that in the message input.

**Comment by koral--** [comment]
Yep, it works. So it seems that there is no need to change anything in the step.
If one don't want unfurling, he can add backticks around a URL.
